src/components/nav.tsx

```diff
@@ -99,7 +120,8 @@ export const Nav: React.FC<{}> = observer(() => {
         const fetchData = async () => {
             const key = await CosmosNodeService.instance.cosmosClient.keplr.getKey(CosmosNodeService.instance.chainInfo.chainId)
             console.log('key', key)
-            setUserString(`${key.name}: ${key.bech32Address}`)
+            let shortAddress = key.bech32Address.substring(0, 9) + "…" + key.bech32Address.substring(key.bech32Address.length - 4)
+            setUserString(`${key.name}: ${shortAddress}`)
```

Comment:
this address is shown for development purposes only (it doesn't present on figma) - sometimes it is handy to copy-paste the full version of address from here

yet, let's leave your variant for now